packages/squiggle-lang/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "squiggle-experimental",
+  "name": "@foretold-app/squiggle",
```

Comment:
Are you sure it needs the "@foretold-app" part? My impression was that the @ typically referred to the monorepo. In this case, the monorepo is all about Squiggle, not Foretold.

It could be `@Squiggle/squiggle-lang` or something.